sdk/servicebus/azure-servicebus/azure/servicebus/_common/auto_lock_renewer.py

```diff
@@ -19,14 +19,16 @@
 
 if TYPE_CHECKING:
     from typing import Callable, Union, Optional
+
     Renewable = Union[ServiceBusSession, ServiceBusReceivedMessage]
-    LockRenewFailureCallback = Callable[[Renewable,
-                                         Optional[Exception]], None]
+    LockRenewFailureCallback = Callable[[Renewable, Optional[Exception]], None]
 
 _log = logging.getLogger(__name__)
 
-SHORT_RENEW_OFFSET = .5 # Seconds that if a renew period is longer than lock duration + offset, it's "too long"
-SHORT_RENEW_SCALING_FACTOR = .75 # In this situation we need a "Short renew" and should scale by this factor.
+SHORT_RENEW_OFFSET = 0.5  # Seconds that if a renew period is longer than lock duration + offset, it's "too long"
+SHORT_RENEW_SCALING_FACTOR = (
+    0.75  # In this situation we need a "Short renew" and should scale by this factor.
+)
```

Comment:
This formatting is a bit odd for a constant.... I think I'd prefer we just moved the comment to the line above - otherwise it looks a bit like a misconstructed tuple.

Reply:
Nevermind - black has used this formatting throughout - so I'll live with it :)